### PR TITLE
Configure NX Workspaces

### PR DESCRIPTION
**Pull request by anthlasserre**

### CODE REVIEW

File: nx.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    "default": {
+      "runner": "@nrwl/nx-cloud",
+      "options": {
+        "cacheableOperations": [
+          "build",
+          "lint",
+          "test",
+          "e2e"
+        ],
+        "accessToken": "NTg2MGE1NTYtNjBkOS00ZTU1LWFkYWEtNWJhOTZjYzJiODcxfHJlYWQtd3JpdGU="
```
Comment (kimak) [inline nx.json:24]: I guess this should not be visible in the repository?  🙈

Do we need @nrwl/nx-cloud in this project?

File: package.json [hunk 2]
```diff
@@ -1,50 +1,31 @@
 {
-  "name": "@getluko/eslint-plugin-mobile-guidelines",
-  "author": "luko",
-  "version": "0.0.1",
-  "description": "Custom ESLint Rules & Config created by mobile team",
-  "main": "dist/",
+  "name": "eslint-getluko-react-native-guidelines",
```
Comment (kimak) [inline package.json:2]:
```suggestion
  "name": "eslint-react-native-guidelines",
```
?

I will rename the repo once we both agree on the name

**Review by kimak — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -1,29 +1,72 @@
...
+  ```
 
-- @getluko/mobile-guidelines/lower-dash-case-test-id
+**[⬆ back to top](#table-of-contents)**
 
-This rule is used to enforce a naming convention on testID props
+**Blogs**
 
-![lower-dash-case-test-id](./docs/lower-dash-case-test-id.png "lower-dash-case-test-id")
+  - [Medium](https://javascriptweekly.com/)
```
[the remark this answers is not in the document]
Comment (anthlasserre) [reply README.md:35]: No no. I just forgot to update the link sorry 😬

File: nx.json (hunk 1 above)
Comment (anthlasserre) [reply nx.json]: Nope finally I managed to use the default runner.

**Review by kimak — APPROVED** [review]

**Pull request merged**